lib/wordpress.js

```diff
@@ -187,9 +187,21 @@ extend( Client.prototype, {
 		this.authenticatedCall( "wp.deletePost", id, fn );
 	},
 
-	getPostType: function( name, fn ) {
-		// TODO: fields
-		this.authenticatedCall( "wp.getPostType", name, function( error, postType ) {
+	getPostType: function( name, fields, fn ) {
+		if ( typeof name !== "string" ) {
+			throw new TypeError("Expected first argument to be a post type (string). Recieved "+ typeof name +" instead");
+		}
```

Comment:
Does that mean you'd like me to add the error checking in the other places or remove it from here? Without it, you get a somewhat cryptic `TypeError: fn is not a function`